std/uni.d

```diff
@@ -7612,7 +7612,7 @@ private:
         }
     }
 
-    void convertToBig() pure @nogc nothrow
+    void convertToBig() pure @nogc nothrow @trusted
```

Comment:
stdx.allocator to the help!

Reply:
Thanks for Your review and helpful hints!
Currently I won't touch function bodies and be focussed on function/struct signatures for -dip1000 compliance.
@wilzbach Please close this PR, it will be integrated in an update for PR #6041 as there are overlaps.